charts/prometheus-yet-another-cloudwatch-exporter/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: yet-another-cloudwatch-exporter
+description: Yace - Yet Another CloudWatch Exporter
+type: application
+version: 0.38.0
+appVersion: "v0.61.2"
+home: https://github.com/nerdswords/helm-charts
```

Comment:
Please update the URLs here.

Reply:
I was planning to update urls and versions as a followup, but I'm happy to do here if you prefer.

Only problem is that the new repo at `quay.io/prometheuscommunity/yet-another-cloudwatch-exporter` doesn't have the older tag `"v0.61.2"`, so I'd need to additionally bump `version` and `appVersion` in this PR.

I don't have a strong preference which way to go.

Reply:
Since the old repo is still availible with the old versions, the chart here should do a clear cut should locate the new location exclusively, including version bumps.